src/Parser/Printer/PrettyPrinter.php

```diff
@@ -199,4 +217,16 @@ private function printAssignment(OutputInterface $output, Assignment $statement,
 
         $output->writeln($indent . $statement->object->relativeName . ' = ' . $statement->value->value);
     }
+
+    private function printNopStatement(OutputInterface $output): void
+    {
+        if($this->prettyPrinterConfiguration->isIncludeEmptyLineBreaks()) {
+            $output->writeln('');
+        }
+    }
+
+    private function closeCondition(bool $hasNext): bool
+    {
+        return !$hasNext || $this->prettyPrinterConfiguration->isAddClosingGlobal();
+    }
```

Comment:
To match my comment for `PrettyPrinterConfiguration.php`:

```suggestion
    private function printNopStatement(OutputInterface $output): void
    {
        if ($this->prettyPrinterConfiguration->shouldIncludeEmptyLineBreaks()) {
            $output->writeln('');
        }
    }

    private function closeCondition(bool $hasNext): bool
    {
        return !$hasNext || $this->prettyPrinterConfiguration->shouldAddClosingGlobal();
    }
```